person.rb

```diff
@@ -0,0 +1,38 @@
+# Create a class person
+export class Person
+    def initialize(id, name, age)
+        @id = id
+        @name = name
+        @age = age
+    end
+    def initialize(name, age, parent_permission = true)
+        @name = name
+        @age = age
+        @parent_permission = parent_permission
+    end
+    # create a getters for @id, @name, and @age
```

Comment:
- [ ] Please make sure you don't have two methods with same name.

Reply:
I follow the project requirements can you check it again, please!?